src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -200,6 +200,17 @@ actionRegistry.registerWorkbenchAction(
 	ToggleQueryResultsKeyboardAction.LABEL
 );
 
+actionRegistry.registerWorkbenchAction(
+	SyncActionDescriptor.create(
+		ToggleFocusBetweenQueryEditorAndResultsAction,
+		ToggleFocusBetweenQueryEditorAndResultsAction.ID,
+		ToggleFocusBetweenQueryEditorAndResultsAction.LABEL,
+		{ primary: KeyMod.WinCtrl | KeyMod.Shift | KeyCode.KEY_N },
```

Comment:
FYI OneNote uses win+shift+N to open itself so that's kind of annoying to deal with as a default. Consider using a different letter if you don't have a specific reason for using N

Reply:
using F key now